src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -396,18 +401,18 @@ const CallTable = ({
                         h(
                           Link,
                           {
-                            'aria-label': 'View stdout logs',
-                            onClick: () => showLogModal(stdout, true),
+                            'aria-label': 'Logs',
```

Comment:
```suggestion
```

Similar to a previous comment, `aria-label` is not necessary here because the link has text content ("Logs") which provides an accessible name.